package exception;

import javax.swing.*;

public class Exception extends Throwable {
    public static void main(String[] args) throws NullException {
//
//        String str = JOptionPane.showInputDialog(null,"Nese daxil edin");
//
//
//        try{
//            int n = Integer.parseInt(str);
//            JOptionPane.showMessageDialog(null,"Tesekkur " + n);
//            if (str == null) throw new NullException(NullException.);
//            if (str == " ") throw new NoNumberException();
//
//        }
//        catch(NullException e){
//                JOptionPane.showMessageDialog(null,"Sehv daxil etdiz");
//                System.out.println(e);
//            }
//        catch (NoNumberException e){
//                JOptionPane.showMessageDialog(null,"Reqem daxil etdin");
//                System.out.println(e);
//            }
//        finally {
//            System.out.println("Proqram tamamlandi");
//
//        }












//        int [] arr = {7,10,12,4,1};
//
//        String str = JOptionPane.showInputDialog(null,"Indeksi daxil edin!");
//
//
//        try{
//            int n = Integer.parseInt(str);
//            if(n<0) throw new NumberFormatException();
//            JOptionPane.showMessageDialog(null,arr[n]);
//        }
//        catch(NumberFormatException e){
//            JOptionPane.showMessageDialog(null,"Reqem daxil etdin");
//            System.out.println(e);
//        }
//        catch (ArrayIndexOutOfBoundsException e){
//            JOptionPane.showMessageDialog(null,"Indeksi sehv daxil etdin");
//            System.out.println(e);
//        }
//        finally {
//            System.out.println("Proqram tamamlandi");
//        }

    }
}
